### PR TITLE
Dev 410 vrs boc

### PR DESCRIPTION
**Pull request by LifeOfBrianOC**

### CODE REVIEW

File: PowerVCF.psm1 [hunk 1]
```diff
@@ -4342,6 +4342,194 @@ Export-ModuleMember -Function Reset-VCFvRSLCM
...
+        .EXAMPLE
+        PS C:\> Get-VCFvROPs
+        This example list all details concerning the vRealize Operations Manager
+
+        .EXAMPLE
+        PS C:\> Get-VCFvROPs -domains
+        This example lists all workload domains connected to vRealize Operations Manager
+    #>
+
+    Param (
```
Comment (GaryJBlake) [inline PowerVCF.psm1:4365]: Is this a formatting issue ?

File: PowerVCF.psm1 [hunk 2]
```diff
@@ -4342,6 +4342,194 @@ Export-ModuleMember -Function Reset-VCFvRSLCM
...
+    	[Parameter (Mandatory=$true)]
+      		[ValidateNotNullOrEmpty()]
+            [string]$domain,
+        [Parameter (Mandatory=$true)]
+            [ValidateSet("ENABLED", "DISABLED")]
+            [ValidateNotNullOrEmpty()]
+            [string]$status
+    )
```
Comment (GaryJBlake) [inline PowerVCF.psm1:4417]: Same here formatting issue for Parameters

File: PowerVCF.psm1 [hunk 3]
```diff
@@ -4342,6 +4342,194 @@ Export-ModuleMember -Function Reset-VCFvRSLCM
...
+
+        .EXAMPLE
+        PS C:\> Get-VCFvRA
+        This example list all details concerning the vRealize Automation
+    #>
+
+    Try {
+        createHeader # Calls createHeader function to set Accept & Authorization
+        checkVCFToken # Calls the CheckVCFToken function to validate the access token and refresh if necessary
+        $uri = "https://$sddcManager/v1/vropses"
```
Comment (GaryJBlake) [inline PowerVCF.psm1:4491]: uri does not seem right to me ?

File: docs/functions/vRealize Log Insight/Set-VCFvRLI.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# Set-VCFvRLI
```
Comment (GaryJBlake) [inline docs/functions/vRealize Log Insight/Set-VCFvRLI.md:1]: Not seeing the Set-VCFvRLI in the main file ?

File: docs/functions/vRealize Log Insight/Set-VCFvRLI.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+- ENABLED or DISABLED
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
```
Comment (GaryJBlake) [inline docs/functions/vRealize Log Insight/Set-VCFvRLI.md:52]: Add Accepted Value: ENABLE,DISABLE ?

File: docs/functions/vRealize Operations Manager/Set-VCFvROPs.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+- ENABLED or DISABLED
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
```
Comment (GaryJBlake) [inline docs/functions/vRealize Operations Manager/Set-VCFvROPs.md:52]: Add Accepted Value: ENABLE,DISABLE ?

**Review by GaryJBlake — CHANGES_REQUESTED** [review]

File: PowerVCF.psm1 (hunk 3 above)
Comment (LifeOfBrianOC) [reply PowerVCF.psm1:4491]: uri is correct i'm afraid

File: docs/functions/vRealize Log Insight/Set-VCFvRLI.md (hunk 4 above)
Comment (LifeOfBrianOC) [reply docs/functions/vRealize Log Insight/Set-VCFvRLI.md:1]: Added in a later commit

**Review by GaryJBlake — APPROVED** [review]
Looks Good

**Pull request merged**